### ISSUE TITLE
Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node 

### ISSUE THREAD

**Issue by Thaslim** [deepmind/graph_nets#94]
Hi,
I am training faster_rcnn_inception_v2_coco_2018_01_28.
I have the following configuration in conda env:
tensorflow: 1.13.1
python 3.6.9
cuda 10.0
I am training on HPC.
 when I run model_main.py, I get the following error, but training is still progressing, but it is too slow.
Any suggestions on solving the issue are much welcome.

2019-11-20 18:23:33.847898: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1512] Adding visible gpu devices: 0
2019-11-20 18:23:33.847969: I tensorflow/core/common_runtime/gpu/gpu_device.cc:984] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-11-20 18:23:33.847989: I tensorflow/core/common_runtime/gpu/gpu_device.cc:990]      0 
2019-11-20 18:23:33.848004: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1003] 0:   N 
2019-11-20 18:23:33.848121: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1115] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 11338 MB memory) -> physical GPU (device: 0, name: Tesla P100-PCIE-12GB, pci bus id: 0000:03:00.0, compute capability: 6.0)
2019-11-20 18:23:35.108870: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node ChangeCoordinateFrame/strided_slice. Error: Pack node (stack) axis attribute is out of bounds: 0
2019-11-20 18:23:35.108942: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node ChangeCoordinateFrame/strided_slice_2. Error: Pack node (stack) axis attribute is out of bounds: 0
2019-11-20 18:23:35.108980: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node ChangeCoordinateFrame/strided_slice_6. Error: Pack node (stack) axis attribute is out of bounds: 0
2019-11-20 18:23:35.109011: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node ChangeCoordinateFrame/strided_slice_7. Error: Pack node (stack) axis attribute is out of bounds: 0
2019-11-20 18:23:35.254310: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node ChangeCoordinateFrame/strided_slice_7. Error: Pack node (stack) axis attribute is out of bounds: 0
2019-11-20 18:23:35.254376: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node ChangeCoordinateFrame/strided_slice_6. Error: Pack node (stack) axis attribute is out of bounds: 0
2019-11-20 18:23:35.254407: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node ChangeCoordinateFrame/strided_slice_2. Error: Pack node (stack) axis attribute is out of bounds: 0
2019-11-20 18:23:35.254435: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node ChangeCoordinateFrame/strided_slice. Error: Pack node (stack) axis attribute is out of bounds: 0
2019-11-20 18:23:35.280847: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node ChangeCoordinateFrame/strided_slice_7. Error: Pack node (stack) axis attribute is out of bounds: 0
2019-11-20 18:23:35.280889: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node ChangeCoordinateFrame/strided_slice_6. Error: Pack node (stack) axis attribute is out of bounds: 0
2019-11-20 18:23:35.280938: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node ChangeCoordinateFrame/strided_slice_2. Error: Pack node (stack) axis attribute is out of bounds: 0
2019-11-20 18:23:35.280977: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node ChangeCoordinateFrame/strided_slice. Error: Pack node (stack) axis attribute is out of bounds: 0
2019-11-20 18:23:35.800968: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node ChangeCoordinateFrame/strided_slice. Error: Pack node (stack_17) axis attribute is out of bounds: 0
2019-11-20 18:23:35.801039: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node ChangeCoordinateFrame/strided_slice_2. Error: Pack node (stack_17) axis attribute is out of bounds: 0
2019-11-20 18:23:35.801072: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node ChangeCoordinateFrame/strided_slice_6. Error: Pack node (stack_17) axis attribute is out of bounds: 0
2019-11-20 18:23:35.801102: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node ChangeCoordinateFrame/strided_slice_7. Error: Pack node (stack_17) axis attribute is out of bounds: 0

**Comment by alvarosg** [comment]
How is the `graph_nets` library used in this example, or what is `model_main.py` referring to?

**Comment by alvarosg** [comment]
(Closing due to inactivity)
